### PR TITLE
Update NMI notation

### PR DESCRIPTION
**Pull request by jeffkause**
#  #

## 🗣 Description ##

Added notation for asterisk in the table.

## 💭 Motivation and context ##

Clarify what the asterisk means in the Ad-hoc reports, pertaining to the BOD 23-02, networked management interface table entry.

## 🧪 Testing ##

Generated sample report.

## ✅ Pre-approval checklist ##

- [X] This PR has an informative and human-readable title.
- [X] Changes are limited to a single goal - *eschew scope creep!*
- [X] All relevant type-of-change labels have been added.
- [X] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [X] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [X] All new and existing tests pass.

## ✅ Post-merge checklist ##

 - [x] Deploy this change to Production

### CODE REVIEW

File: cyhy_report/cyhy_notification/notification.mustache [hunk 1]
```diff
@@ -172,7 +172,7 @@ To further help prioritize remediation decisions, please take note of any ``rans
 
 Cyber Hygiene scans of your host(s) conducted in the past day have detected new potentially risky services.
 
-These services warrant your attention.  All services are potentially at risk of attack, but some can be more risky when open to the public (e.g. RDP, Telnet, etc.), especially if they are open as Networked Management Interfaces.  CISA recommends validating that each service below is intended to be available to the public and, where applicable, the service is up-to-date on the latest version, correctly configured, and uses strong authentication.
+These services warrant your attention.  All services are potentially at risk of attack, but some can be more risky when open to the public (e.g. RDP, Telnet, etc.), especially if they are open as Networked Management Interfaces.  CISA recommends validating that each service below is intended to be available to the public and, where applicable, the service is up-to-date on the latest version, correctly configured, and uses strong authentication. \textbf{\color{cisa-red} *} \color{black}  Denotes the possibility of a networked management interface.
```
Comment (jsf9k) [inline cyhy_report/cyhy_notification/notification.mustache]: I think this is slightly clearer.
```suggestion
These services warrant your attention.  All services are potentially at risk of attack, but some can be more risky when open to the public (e.g. RDP, Telnet, etc.), especially if they are open as Networked Management Interfaces.  CISA recommends validating that each service below is intended to be available to the public and, where applicable, the service is up-to-date on the latest version, correctly configured, and uses strong authentication. A red asterisk (\textbf{\color{cisa-red} *} \color{black}) denotes the possibility of a networked management interface.
```

**Review by jsf9k — APPROVED** [review]
I have a minor suggestion, which you may take or leave as you see fit.

**Comment by jsf9k** [comment]
@jeffkause - When you accept someone's suggested change please change the default commit message to something more descriptive of what is actually being changed.  In the case of commit b9169ba, something like "Tweak wording for clarity" would be a vast improvement over "Update cyhy_report/cyhy_notification/notification.mustache".

**Comment by jeffkause** [comment]
Shane,

Thank you for the feedback. I am still learning the ropes. Any feedback or help is truly appreciated.
Jeff
________________________________

**Review by dav3r — APPROVED** [review]
*️⃣ 👍

**Comment by dav3r** [comment]
@jeffkause Please note that I added a "Post-merge checklist" item above so that we don't forget to deploy this change.

**Comment by jeffkause** [comment]
[like]  Kause, Jeffrey (CTR) reacted to your message:
________________________________

**Pull request merged**